qiita_ware/analysis_pipeline.py

```diff
@@ -26,7 +27,11 @@ def _build_analysis_files(analysis, r_depth=None, **kwargs):
     analysis : Analysis object
         The analysis to build files for
     r_depth : int, optional
-        Rarefaction depth for biom table creation. Default None
+        Rarefaction depth for biom table creation. Default
```

Comment:
I think this line got cut. `Default: None`. Also, it was not there before but I think this will improve the documentation:
`Rarefaction depth for biom table creation. Default: None, no rarefaction is applied`. If I understand correctly, that is what is happening - if my description of `None` is off, then is even more important to have the docs.